index.html

```diff
@@ -5,24 +5,27 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Mens Portfolio</title>
+        
         <link rel="stylesheet" href="css/index.css">
         <link rel="stylesheet" href="css/works.css">
         <link rel="stylesheet" href="css/about.css">
         <link rel="stylesheet" href="css/contact.css">
+        <link rel="stylesheet" href="css/desktop.css">
+        
     </head>
     <body>
 
         <div class="flex_container">
             <header>
-                <div class="mobile_logo"></div>
-                <nav class="flex_nav">
-                    <a href="" class="toggleNav"> 
+                <div class="mobile_logo mobile_off"></div>
+                <nav class="flex_nav  text_body_font text_color">
+                    <a href="" class="toggleNav mobile_off"> 
                         <div class="bar"></div>

```

Comment:
- [ ] Kindly check your mobile version because there's an overflow in your design, Kindly fix it.

![image](https://user-images.githubusercontent.com/35535130/160386266-d26872d6-95ca-4dff-a9fb-d68dc2b0f670.png)